include/libdnf/comps/comps.hpp

```diff
@@ -17,15 +17,14 @@ You should have received a copy of the GNU Lesser General Public License
 along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 */
 
+
```

Comment:
I think we only use one linebreak in this place (between copyright notice and includes), though we don't have this written anywhere...

Reply:
I thought it's the other way around, because there are double linebreaks in some places. I changed it.